ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/panelcontainer/v1/panelcontainer/simple.html

```diff
@@ -24,8 +24,10 @@
          class="cmp-container${wcmmode.edit ? ' {0}': '' @ format=[allowed.cssClass]}"
          title="${panel.tooltipVisible ? '' : panel.tooltipText}"
          role="${container.roleAttribute}"
-         style="${container.backgroundStyle @ context='styleString'}">
-        <div data-sly-call="${label.label @componentId=panel.id, labelValue=panel.label.value, labelVisible=panel.label.visible, bemBlock='cmp-container'}" data-sly-unwrap></div>
+         style="${container.backgroundStyle @ context='styleString'}"
+         data-sly-test.widgetId="${'{0}-{1}' @ format=[panel.id, 'widget']}"
+         aria-labelledby="${panel.label.visible != false ? widgetId : ''}">
+        <div data-sly-call="${label.label @componentId=widgetId, labelValue=panel.label.value, labelVisible=panel.label.visible, bemBlock='cmp-container', withId=true}" data-sly-unwrap></div>
```

Comment:
I don't see the value of withId set to false anywhere ? Is it true always

Reply:
the default value of withId is false